speech/google/cloud/speech/sample.py

```diff
@@ -69,6 +71,15 @@ def __init__(self, content=None, source_uri=None,
             raise ValueError('Invalid encoding: %s' % (encoding,))
 
     @property
+    def chunk_size(self):
+        """Chunk size to send over GRPC. ~100ms
+
+        :rtype: int
+        :returns: Optimized chunk size.
+        """
+        return int(self.sample_rate / 10)
```

Comment:
@dhermes I think 1599 would be fine in that case for a chunk size.

Is the request still to use `/` instead of `//` and divide by 10.0? What's the benefit?